src/js/edu-benefits/1990-rjsf/config/form.js

```diff
@@ -24,20 +26,18 @@ const formConfig = {
   title: 'Apply for education benefits',
   subTitle: 'Form 22-1990',
   chapters: {
-    veteranInformation: {
-      title: 'Veteran Information',
+    applicantInformation: {
+      title: 'Applicant Information',
       pages: {
-        veteranInformation: {
-          title: 'Veteran information',
-          path: 'veteran-information',
-          uiSchema: {
-          },
-          schema: {
-            type: 'object',
-            properties: {
-            }
-          }
-        }
+        applicantInformation: applicantInformation(fullSchema1990, {
+          isVeteran: true,
+          fields: [
+            'veteranFullName',
+            'veteranSocialSecurityNumber',
+            'veteranDateOfBirth',
```

Comment:
I think there's some extra validation around date of birth, where you have to be over 17. Look in `src/js/edu-benefits/1990/utils/validations.js`.